loadgen/README.md

```diff
@@ -103,3 +99,67 @@ with the reference models.
 
 For templates of how to do the above in detail, refer to code for the demos,
 tests, and reference models.
+
+
+## Submissions over the network
+
+For reference, on a high level a submission looks like this:
+
+<div align="center" style="display:flex; flex-flow:row wrap; justify-content: space-evenly;">
+<img src="diagram_submission.png" width="300px" style="padding: 20px">
+</div>
+
+The loadgen implementation is common to all submissions, while the QSL (“Query Sample Library”) and SUT (“System Under Test”) are implemented by submitters. QSL is responsible for loading the data and includes untimed preprocessing.
+
+A submission over the network introduces a new component “QDL” (query dispatch library) and a it would look like this:
+
+<div align="center" style="display:flex; flex-flow:row wrap; justify-content: space-evenly;">
+<img src="diagram_network_submission.png" width="300px" style="padding: 20px">
+</div>
+
+QDL is a proxy for a load-balancer, that dispatches queries to SUT over a physical network, receives the responses and passes them back to loadgen.  It is implemented by the submitter. The interface of the QDL is the same as the API to SUT. 
+
+In scenarios using QDL, data in QSL may be compressed at the choice of the submitter in order to reduce network transmission time. Decompression is part of the timed processing in SUT. A set of approved standard compression schemes will be specified for each benchmark; additional compression schemes must be approved in advance by the Working Group.
+
+All communication between loadgen/QSL and SUT is via QDL, and all communication between QDL and SUT must pass over a physical network.
+
+Query batching can be implemented either in QDL or SUT, or split between the two components, at the submitter’s option.  Current Plan of record is no batching in QDL. QDL batching would be considered for future releases. Aside from batching, QDL implements only the protocol to transmit queries over the network and receive responses. Performing any part of the timed preprocessing or inference in QDL is specifically disallowed.
+
+The MLperf over the Network will run in Server mode and Offline mode. All loadgen modes are expected to work as is with insignificant changes. These include running the test in performance mode, accuracy mode, find peak performance Search mode and compliance mode. The same applies for power measurements.
```

Comment:
@pgmpablo157321 What is the "find peak performance Search mode"?

Reply:
I assume is the `FindPeakPerformance` mode defined here:
https://github.com/mlcommons/inference/blob/de6497f9d64b85668f2ab9c26c9e3889a7be257b/loadgen/test_settings.h#L99-L104
I'll remove the 'Search' it to reflect this mode better